src/inmanta/main.py

```diff
@@ -801,6 +801,65 @@ def bootstrap_token(client: Client) -> None:
     click.echo("Token: " + protocol.encode_token(["api", "compiler", "agent"], expire=3600))
 
 
+@cmd.group("action-log", help="Subcommand to view the resource action log")
+@click.pass_context
+def resource_action_log(ctx: click.Context) -> None:
+    pass
+
+
+def validate_resource_version_id(
+    ctx: click.Context, option: Union[click.Option, click.Parameter], value: Any
+) -> ResourceVersionIdStr:
+    if not Id.is_resource_version_id(value):
+        raise click.BadParameter(value)
+    return value
+
+
+@resource_action_log.command(name="list")
+@click.option("--environment", "-e", help="The ID or name of the environment to use", required=True)
+@click.option(
+    "--rvid", help="The resource version ID of the resource", callback=validate_resource_version_id, required=True,
+)
+@click.option("--action", help="Only list this resource action", type=click.Choice([ra.value for ra in ResourceAction]))
+@click.pass_obj
+def resource_action_log_list(client: Client, environment: str, rvid: ResourceVersionIdStr, action: Optional[str]) -> None:
+    tid = client.to_environment_id(environment)
+    """
+        List the resource action log for a specific Resource.
+    """
```

Comment:
This looks like the docstring for `resource_action_log_list`. Should it be one line up?

Reply:
Indeed. This should be the docstring.